<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PUJJMK1" failures="1" tests="2" name="Test" time="3.164" errors="0" timestamp="2024-03-12T10:48:55 IST">
  <testcase classname="com.scenario.automate.RestApiTest" name="restApiTestOne" time="2.192"/>
  <testcase classname="com.scenario.automate.RestApiTest" name="restApiTestTwo" time="0.914">
    <failure type="java.lang.AssertionError" message="expected [5] but found [6]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [6]
at com.scenario.automate.RestApiTest.restApiTestTwo(RestApiTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- restApiTestTwo -->
</testsuite> <!-- Test -->
